docs/Installing.asciidoc

```diff
@@ -311,6 +310,23 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Needle Caching  
+~~~~~~~~~~~~~~~
+
+If you your network is slow or you experience long time to load needles you
```

Comment:
Well the double you is there for a reason :), a comma might help... 

> If you your network is slow or you experience long time to load needles, you might want to consider needle caching.
We can also use:
> If you your network is slow or needles are taking too long to load, you might want to consider needle caching.